gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -71,6 +115,7 @@ def run(
                 or ill-constrained system).
             A GtsfmMetricsGroup of 1DSfM metrics.
         """
+        logger.info("Running translation averaging")
```

Comment:
Is there a reason for adding this?

Reply:
was debugging locally, but I realized we need better logging statements. I think Ill create a PR where I just add and remove some logging statements across the pipeline.